gcloud/bigtable/row_filters.py

```diff
@@ -119,7 +119,7 @@ class _RegexFilter(RowFilter):
     """
 
     def __init__(self, regex):
-        self.regex = regex
+        self.regex = _to_bytes(regex)
```

Comment:
@tseaver this has moved after #1602.

Previous concern was that `_to_bytes` should be used in `to_pb` and I remarked that it would re-do work every time `to_pb` was called.